src/explore-education-statistics-frontend/src/modules/data-catalogue/components/__tests__/DownloadStep.test.tsx

```diff
@@ -0,0 +1,259 @@
+    expect(downloads[0]).toEqual(
+      DownloadsGroup.getByLabelText('Another Subject (csv, 101mb)'),
+    );
+
+    const subject2Hint = within(getDescribedBy(container, downloads[0]));
+    expect(
+      within(subject2Hint.getByTestId('Content')).getByText(
+        'Some content here 2',
+      ),
+    ).toBeInTheDocument();
+    expect(subject2Hint.getByTestId('Geographic levels')).toHaveTextContent(
+      'WeYo',
+    );
+    expect(subject2Hint.getByTestId('Time period')).toHaveTextContent(
+      '2016 to 2019',
+    );
+
+    expect(downloads[1]).toHaveAttribute('value', 'file-1');
+    expect(downloads[1]).toBeEnabled();
+    expect(downloads[1]).not.toBeChecked();
+    expect(downloads[1]).toEqual(
+      DownloadsGroup.getByLabelText('Subject 1 (csv, 100mb)'),
+    );
+    const subject1Hint = within(getDescribedBy(container, downloads[1]));
+    expect(
+      within(subject1Hint.getByTestId('Content')).getByText(
+        'Some content here 1',
+      ),
+    ).toBeInTheDocument();
+    expect(subject1Hint.getByTestId('Geographic levels')).toHaveTextContent(
+      'SoYo',
+    );
+    expect(subject1Hint.getByTestId('Time period')).toHaveTextContent(
+      '2018 to 2019',
+    );
+
+    expect(downloads[2]).toHaveAttribute('value', 'file-3');
+    expect(downloads[2]).toBeEnabled();
+    expect(downloads[2]).not.toBeChecked();
+    expect(downloads[2]).toEqual(
+      DownloadsGroup.getByLabelText('Subject 3 (csv, 100mb)'),
+    );
+
+    const subject3Hint = within(getDescribedBy(container, downloads[2]));
+    expect(
+      within(subject3Hint.getByTestId('Content')).getByText(
+        'Some content here 3',
+      ),
+    ).toBeInTheDocument();
+    expect(subject3Hint.getByTestId('Geographic levels')).toHaveTextContent(
+      'NoYo',
+    );
+    expect(subject3Hint.getByTestId('Time period')).toHaveTextContent(
+      '2017 to 2019',
+    );
+
+    expect(
+      screen.getByRole('button', { name: 'Download selected files' }),
+    ).toBeInTheDocument();
+  });
+
+  test('shows latest data tag', () => {
+    render(
+      <DownloadStep
+        subjects={testSubjects}
+        isLatestRelease
+        onSubmit={noop}
+        {...wizardProps}
+      />,
+    );
+
+    expect(screen.getByText('This is the latest data')).toBeInTheDocument();
+  });
+
+  test('shows not latest data tag', () => {
+    render(
+      <DownloadStep
+        subjects={testSubjects}
+        isLatestRelease={false}
+        onSubmit={noop}
+        {...wizardProps}
+      />,
+    );
+
+    expect(screen.getByText('This is not the latest data')).toBeInTheDocument();
+  });
+
+  test('renders a message when there are no downloads', () => {
+    render(
+      <DownloadStep
+        subjects={[]}
+        isLatestRelease={false}
+        onSubmit={noop}
+        {...wizardProps}
+      />,
+    );
+
+    expect(
+      screen.queryByRole('group', {
+        name: /Choose files to download/,
+      }),
+    ).not.toBeInTheDocument();
+
```

Comment:
Might want to check that they cannot submit as well?

Reply:
The button isn't shown when there are no files (following the same pattern as the table tool), so I've tested for that.